src/dialog-user/services/dialogData.ts

```diff
@@ -108,10 +90,12 @@ export default class DialogDataService {
     }
 
     if (data.type === 'DialogFieldDateControl' || data.type === 'DialogFieldDateTimeControl') {
-      defaultValue = data.dateField ? new Date(data.dateField) : new Date();
+      defaultValue = data.default_value ? new Date(data.default_value) : new Date();
     }
 
-    if (data.type === 'DialogFieldDropDownList' && data.options.force_multi_value && data.default_value) {
+    // FIXME maybe better make sure it's never not string (must come from double call)
```

Comment:
Maybe putting this conditional into a private method with a more descriptive name would help?

Reply:
I just don't like that we're potentially converting twice, but I think it will need a bit more refactoring to make sure we only run the setup once.

So.. leaving for another PR if you don't mind :)